packages/ngtools/webpack/src/ngcc_processor.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { JsonParseMode, parseJson } from '@angular-devkit/core';
+import * as ts from 'typescript';
+import { time, timeEnd } from './benchmark';
+import { WebpackCompilerHost } from './compiler_host'; // tslint:disable-line:no-circular-imports
+import { workaroundResolve } from './utils';
+
+export class NgccProcessor {
+  private _ngcc: typeof import('@angular/compiler-cli/ngcc') | undefined;
+  private _processedModules = new Set<string>();
+
+  constructor(
+    private readonly propertiesToConsider: string[],
+    private readonly host: WebpackCompilerHost,
+  ) {
+    // this is done for the sole reason that @ngtools/webpack
+    // support versions of Angular that don't have NGCC API
+    try {
+      this._ngcc = require('@angular/compiler-cli/ngcc');
+    } catch {
+    }
+  }
+
+  processModule(
+    moduleName: string,
+    resolvedModule: ts.ResolvedModule | ts.ResolvedTypeReferenceDirective,
+  ): void {
+    const resolvedFileName = resolvedModule.resolvedFileName;
+    if (
+      !this._ngcc
+      || !resolvedFileName
+      || moduleName.startsWith('.')
```

Comment:
what about `startsWith('/')`?

Reply:
That's actually a good point, we should't process absolute modules, that said, `/` alone is not enough for Windows users.